FlexateWebApi/Controllers/HealthChecksController.cs

```diff
@@ -15,7 +15,7 @@ public class HealthChecksController : ControllerBase
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
-        public async Task<ActionResult<string>> GetAsync()
```

Comment:
this comment is for like 10 :) wouldn't it be better design-wise to name api route "api/healthchecks" so it's more clear and understandable without looking at class name?